### PR TITLE
feat: add tertiary button component

### PR DESCRIPTION
**Pull request by paul-corbett-qantas**
## Description

[](https://app.shortcut.com/qantasaccommodation/story/78312/tertiary-button)

💁‍
Added a new component TertiaryButton and the requested variants (align = 'left' | 'center', disabled = true | false)

image

## Related issues

💪

## Looks like this

🤨
image

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan) [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [x] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/TertiaryButton/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { TertiaryButton } from 'roo-ui/components';
+
+export default <TertiaryButton>Hello World</TertiaryButton>;
+```
+
+## Properties
+
+| Name       | Description           | Type             | Default | Required? |
+| :--------- | :-------------------- | :--------------- | :------ | :-------- |
+| `align`    | align the button text | `left` / `right` | `left`  | -         |
```
Comment (klubiniecki-qantas) [inline src/components/TertiaryButton/README.md:23]: Why not left / right / center? If we're doing the alignments, that should be easy to add.

File: src/components/TertiaryButton/TertiaryButton.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  &:disabled {
+    color: ${themeGet('colors.greys.dusty')};
+    cursor: not-allowed;
+  }
+`;
+
+TertiaryButton.propTypes = {
+  align: 'left' | 'center',
```
Comment (philipwindeyer) [inline src/components/TertiaryButton/TertiaryButton.js:34]: README says left or right, but prop only allows for left or center?

**Review by philipwindeyer — CHANGES_REQUESTED** [review]
LGTM aside from the `align` options

File: src/components/TertiaryButton/README.md (hunk 1 above)
Comment (ryangaoqantashotels) [reply src/components/TertiaryButton/README.md:23]: I think it should only be left or center, I'll update it

File: src/components/TertiaryButton/TertiaryButton.js (hunk 2 above)
Comment (ryangaoqantashotels) [reply src/components/TertiaryButton/TertiaryButton.js:34]: I think it should only be left or center, I'll update README

File: src/components/TertiaryButton/README.md (hunk 1 above)
Comment (klubiniecki-qantas) [reply src/components/TertiaryButton/README.md]: Please double check with Dani or Kym, doesn't make sense to do the work twice if it's needed. Other than that all good 🚀

**Review by klubiniecki-qantas — APPROVED** [review]

**Review by philipwindeyer — APPROVED** [review]
🚀  LGTM

**Pull request merged**